src/main/kotlin/org/openmbee/flexo/sysmlv2/apis/ElementApi.kt

```diff
@@ -161,7 +114,7 @@ fun FlexoModelHandler.extractModelElementToJson(elementIri: String): JsonObject
                 val obj = values.elementAt(0)
 
                 // properties
-                if(predicate.uri.startsWith(SYSMLV2.SYSML)) {
+                if(predicate.uri.startsWith(SYSMLV2.VOCABULARY)) {
```

Comment:
this is back to prop

Reply:
also the propertyKey being the iri.suffix only worked if it's the vocabulary, otherwise i had to split on : and get the last string

Reply:
need to handle relation too at line 100 since it wasn't used before

Reply:
fixed and removed some un-needed parts